recipe/meta.yaml

```diff
@@ -12,13 +12,17 @@ source:
     - CMakeLists.txt.patch  # [win]
     - config.h.patch  # [win]
     - configure.cmake.patch  # [win]
+    - utf_8_mac.patch  # [osx]
```

Comment:
Can you add the origin of this patch?

Reply:
I made it by hand diffing on my own machine. I will add a comment.